src/components/Settings/MultiServiceContainerCard.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import type { Screen } from "@/router/helpers/types";
+
+import { ScrollView, Image, Text, View } from "react-native";
+import LottieView from "lottie-react-native";
+import { NativeItem, NativeList, NativeText } from "../Global/NativeComponents";
+import { LinearGradient } from "expo-linear-gradient";
+import BetaIndicator from "../News/Beta";
+
+const MultiServiceContainerCard = ({ theme }: { theme: any }) => {
+  const { colors } = theme;
```

Comment:
Supprime les import/variables inutilisées

```suggestion
import React, { useEffect, useRef } from "react";

import { Image, View } from "react-native";
import LottieView from "lottie-react-native";
import { NativeItem, NativeList, NativeText } from "../Global/NativeComponents";
import { LinearGradient } from "expo-linear-gradient";
import BetaIndicator from "../News/Beta";

const MultiServiceContainerCard = ({ theme }: { theme: any }) => {
```